docs/setup_visualstudio.md

```diff
@@ -10,338 +10,264 @@ Setting up Visual Studio
+</div>
 
 <img src="images/visualstudio060.png" width="512px" />
 
-Select "Console application project" and then "Finish".
+You can see the files created by Visual Studio in the File Explorer.  Right-click `p1-stats`, select "Show in Finder".
+
+| <img src="images/visualstudio070.png" height="512px" /> | <img src="images/visualstudio071.png" height="512px" /> |
 
-<img src="images/visualstudio065.png" width="512px" />
+<img src="images/visualstudio072.png" width="768px" />
 
+You can also see the files created by Visual Studio from the WSL command line.  Notice that Visual Studio created `.sln`, and `vcxproj` files.
 
-Quit Visual Studio, open a terminal, and navigate to your source code directory (remember the `cd` command).  You'll notice that Visual Studio created a new directory (`.vs/p1-stats.xcodeproj/`) and new files (`p1-stats.sln`, `p1-stats.vcxproj`, `p1-stats.vcxproj.filters`).
 ```console
-$ cd /mnt/c/Users/awdeorio/Desktop/eecs280/p1-stats  # where awdeorio stored his source code
-$ pwd
-/mnt/c/Users/awdeorio/Desktop/eecs280/p1-stats
-$ tree -a -L 2
+$ tree
 .
-├── .vs
-│   └── p1-stats
-├── main_test.in
-├── main_test.out.correct
-├── main_test_data.tsv
-├── Makefile
-├── p1_library.cpp
-├── p1_library.h
+├── p1-stats.cpp
 ├── p1-stats.sln
 ├── p1-stats.vcxproj
 ├── p1-stats.vcxproj.filters
-├── stats.h
-├── stats_public_test.cpp
-└── stats_tests.cpp
+└── p1-stats.vcxproj.user
 ```
 
+### Rename main file
+{: .primer-spec-toc-ignore }
+Rename the default `p1-stats.cpp` to `main.cpp`.  Your main filename may be different.  Use Visual Studio to rename a file, not the command line or File Explorer.
 
-## Add existing files
+Right-click `p1-stats.cpp` and select "Rename".  Change the file name.
 
-Start Visual Studio and open your project.  Here's how to do that all at once, from the command line:
-```console
-$ cmd.exe /c start p1-stats.vcxproj
-```
+| <img src="images/visualstudio075.png" width="256px" /> | <img src="images/visualstudio076.png" width="256px" /> | <img src="images/visualstudio077.png" width="256px" /> |
 
-All our existing files were already added when we selected "Project From Existing Code".  You should see them in the side bar.
-
-<img src="images/visualstudio070.png" width="768px" />
+### Add new files
+<div class="primer-spec-callout info" markdown="1">
+Visual Studio created `main.cpp` by default.  Skip this subsection your first time through the tutorial.  You can come back to it.
+</div>
 
-To add an existing file after the project has been created, right click "Source Files", then select "Add" > "Existing Item".
+Open your project folder by selecting `File` > `Open` > `Project/Solution`, navigate to your project directory (`p1-stats` in this example) and open `p1-stats.sln`.
 
-<img src="images/visualstudio075.png" width="768px" />
+<div class="primer-spec-callout info" markdown="1">
+**Pro-tip:** Here's a quick way to open Visual Studio to a specific project folder from the command line.  FIXME FIXME FIXME
```

Comment:
FIXME :)

Reply:
After a closer look, I'm not seeing a straight-forward, canonical way to do this.  I'm going to cut it.